client/my-sites/people/people-invite-details/index.jsx

```diff
@@ -36,21 +38,59 @@ class PeopleInviteDetails extends React.PureComponent {
 		page.back( fallback );
 	};
 
+	handleDelete = () => {
+		const { deleteInvite: _deleteInvite, deleting, invite, site } = this.props;
+		if ( deleting ) {
+			return;
+		}
+		_deleteInvite( site.ID, invite.key );
+	};
+
+	renderClearOrRevoke = () => {
+		const { deleting, invite, translate } = this.props;
+		const { isPending } = invite;
+		const revokeMessage = translate(
+			'Revoking an invite will no longer allow this person to join your site. ' +
+				'You can always invite them again if your change your mind.'
+		);
+		const clearMessage = translate(
+			'If you no longer wish to see this record, you can clear it. ' +
+				'The person will still remain a member of this site.'
+		);
+
+		return (
+			<div className="people-invite-details__clear-revoke">
+				<div>{ isPending ? revokeMessage : clearMessage }</div>
+				<Button
+					busy={ deleting }
+					primary={ isPending }
+					scary={ isPending }
+					onClick={ this.handleDelete }
+				>
+					{ isPending ? translate( 'Revoke Invite' ) : translate( 'Clear Invite' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Invite' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).